utils/rocvideodecode/roc_video_dec.cpp

```diff
@@ -377,7 +377,7 @@ int RocVideoDecoder::HandleVideoSequence(RocdecVideoFormat *p_video_format) {
     videoDecodeCreateInfo.target_width = target_width_;
     videoDecodeCreateInfo.target_height = target_height_;
 
-    chroma_height_ = (int)(ceil(disp_height_ * GetChromaHeightFactor(video_surface_format_)));
+    chroma_height_ = (int)(ceil(target_height_ * GetChromaHeightFactor(video_surface_format_)));
```

Comment:
chroma_height_ is also set in re-configure call. If we change its calculation here, we need to change in re-configure too.

chroma_height_ is loosely defined and used in quite a few places. This is a good opportunity to review its define and usage in call cases.

Reply:
Yes, please check all the places where output dimensions are calculated and set with and without crop